### ISSUE TITLE
Gradient maker: dragging the line for custom gradients causes excessive flashing

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#208]
Hard to measure but this probably fails the 3 flashes a second rule (and is generally an uncomfortable experience)

Works fine in Chrome, flashes in **Edge**

Steps:
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/svgradientbackgroundmaker/

* Click the "custom" gradient direction
* Drag the diagonal line to make a custom gradient

Expected:

Smooth update of gradient (Chrome)

Actual:

Flashes as it redraws for every point that the line is dragged.

Perhaps a browser bug, but as it is an old demo, I'm not sure if there is a more efficient way of redrawing?
